.github/workflows/release-please.yml

```diff
@@ -35,55 +37,94 @@ jobs:
       hashes: ${{ steps.full-release.outputs.hashes }}
     steps:
       - uses: actions/checkout@v4
+
+      - uses: ./.github/actions/setup-env
+        with:
+          workspace_path: LaunchDarkly.ServerSdk
+
       - uses: ./.github/actions/full-release
         id: full-release
         with:
-          workspace_path: 'pkgs/sdk/server'
-          project_file: 'pkgs/sdk/server/src/LaunchDarkly.ServerSdk.csproj'
-          test_project_file: 'pkgs/sdk/server/test/LaunchDarkly.ServerSdk.Tests.csproj'
-          build_output_path: 'pkgs/sdk/server/src/bin/Release/'
-          dll_name: 'LaunchDarkly.ServerSdk.dll'
-          dry_run: false
+          workspace_path: ${{ env.WORKSPACE_PATH }}
+          project_file: ${{ env.PROJECT_FILE }}
+          build_output_path: ${{ env.BUILD_OUTPUT_PATH }}
+          test_project_file: ${{ env.TEST_PROJECT_FILE }}
+          dll_name: ${{ env.BUILD_OUTPUT_DLL_NAME }}
+          dry_run: ${{ inputs.dry_run }}
           aws_role: ${{ vars.AWS_ROLE_ARN }}
           token: ${{ secrets.GITHUB_TOKEN }}
 
-  release-sdk-server-provenance:
```

Comment:
For reviewers: Moved provenance tasks to the bottom for organization.  Made the diff weird.